docs/specs/clients/sign/data-structures.md

```diff
@@ -85,7 +85,8 @@ Proposal is sent by the proposer client to be approved or rejected by the respon
       "events": [string]
     }
   },
-  "pairingTopic": string
+  "pairingTopic": string,
+  "expiryTimestamp": number, // OPTIONAL timespamp in seconds
```

Comment:
We should add expiryTimestamp on json rpc section for session proposal and session request